mono/metadata/object.c

```diff
@@ -2585,6 +2585,7 @@ mono_remote_class (MonoDomain *domain, MonoStringHandle class_name, MonoClass *p
 	gpointer* key, *mp_key;
 	char *name;
 	
+	g_assert (error);
```

Comment:
This is not useful, since we are going towards an initialization where we allocate them and at API boundaries.

Reply:
agreed